src/config/navigation/defaultStackConfig.tsx

```diff
@@ -153,7 +153,13 @@ export const defaultStackConfig = ({
     },
     {
       routeName: ScreenName.Form,
-      screenComponent: FormScreen
+      screenComponent: FeedbackScreen,
+      screenOptions: { title: texts.screenTitles.feedback }
```

Comment:
this is kept for backwards compatibility, for cases where "FORM" is used to navigate?

Reply:
yes